tests/AutoReview/FixerFactoryTest.php

```diff
@@ -307,19 +304,28 @@ public function testFixerWithNoneDefaultPriorityIsTested(): void
         $missing = [];
 
         foreach ($fixers as $fixer) {
+            if ($fixer instanceof DeprecatedFixerInterface) {
```

Comment:
why? if deprecated fixer is still present, if there is any priority dependency, it should be tested